### DISCUSSION TITLE
Bookmark Group please🙏

### DISCUSSION THREAD

**Discussion by chfly2000** [alefragnani/vscode-bookmarks#480] [Ideas]
It would be a bit messy If I have many bookmarks with many files. Is it possible to divide them into different bookmark group folders? Thank you for consideration.

**Comment by alefragnani** [comment]
Hi @chfly2000,

There is already an issue to track this request (#254). 

Feel free to upvote/comment.

Hope this helps

**Reply by chfly2000** [reply]
@alefragnani Thanks. Looking forward to this great feature!
